adyenv6b2ccheckoutaddon/acceleratoraddon/web/webroot/_ui/responsive/common/js/adyen.checkout.js

```diff
@@ -263,6 +263,76 @@ var AdyenCheckoutHybris = (function () {
             }
         },
 
+        initiatePaypal: function (amount) {
+            var paypalNode = document.getElementById('adyen_hpp_paypal_container');
+
+            var paypalComponent = this.checkout.create("paypal", {
+                environment: "test", // Change this to "live" when you're ready to accept live PayPal payments
+                countryCode: "NL", // Only needed for test. This will be automatically retrieved when you are in production.
+                amount: {
+                    currency: amount.currency,
+                    value: amount.value
+                },
+                intent: "capture", // Change this to "authorize" if the payments should not be captured immediately. Contact Support to enable this flow.
```

Comment:
this configuration is in backoffice. we should set it according to that.

Reply:
Done